src/d2d1.rs

```diff
@@ -0,0 +1,1137 @@
+// Copyright © 2015, Connor Hilarides
+// Licensed under the MIT License <LICENSE.md>
+//! Mappings for the contents of d2d1.h, d2dbasetypes.h, and dcommon.h
+
+use std::ops::Mul;
+use std::default::Default;
+
+pub const D2D1_DEFAULT_FLATTENING_TOLERANCE: ::FLOAT = 0.25;
+
+#[repr(C)]
+#[derive(Clone, Copy, Debug)]

```

Comment:
Could you put the `#[repr]` and `#[derive]` on one line everywhere to save space?